go.sh

```diff
@@ -176,7 +176,7 @@ for t in ${TARGETS}; do
 	rm -rf ${CT_PREFIX}/*/newlib-nano
```

Comment:
We should just set `CT_NEWLIB_NANO_INSTALL_IN_TARGET=n` and use the separate `newlib-nano` directory as done in Keith's original implementation. That seems to be a much cleaner approach IMO.

Reply:
This was evaluated in #379, but this does not seem feasible for the time being (see https://github.com/zephyrproject-rtos/sdk-ng/pull/379#issuecomment-889252433).

Reconsider switching to #379 in the future when the target library stripping patch is updated to handle stripping the libraries under the `newlib-nano` directory.